### ISSUE TITLE
Add --version | -v support to markoc

### ISSUE THREAD

**Issue by austinkelleher** [marko-js/marko#532]
```bash
markoc --version
```

See: #487

**Comment by patrick-steele-idem** [comment]
Currently, `markoc` resolves the Marko compiler from `process.cwd()` so that it resolves the project-specific Marko compiler from the current project:
https://github.com/marko-js/marko/blob/346797a89db9cacef682d92c69efe90b125bff61/bin/markoc.js#L10

The reason I mention that is because should `markoc` should probably report the version of the resolved marko compiler and not the version of the package that the `markoc` resides in.

Are you interested in working on this @austinkelleher ?

**Comment by austinkelleher** [comment]
Sounds good, @patrick-steele-idem .

**Comment by austinkelleher** [comment]
Another note, `--vdom` uses the alias `-v`. Should we change that and make `--version` use the alias `-v`? It would be a breaking change, but `--vdom` was not available in Marko v3, so we don't have to worry about backwards compatibility.

**Comment by patrick-steele-idem** [comment]
@austinkelleher Let's change `--vdom -v` to  `--vdom -V` and introduce `--version -v`. Sound good?

**Comment by austinkelleher** [comment]
We decided that the `markoc -v` will print out two versions. The first being the actual `markoc` version, and the second being the version of Marko that will compile the templates. `markoc template.marko` resolves the Marko from your present working directory, so if you have Marko installed in your project, `markoc` will use the locally installed version of Marko for compiling. It will print out like this:

```bash
# markoc -v
markoc@4.0.0-rc.2
marko@4.0.0-rc.3
```

**Comment by mlrawlings** [comment]
Should we label the versions?

**Comment by patrick-steele-idem** [comment]
@mlrawlings what do you mean by "Should we label the versions?" ?

**Comment by mlrawlings** [comment]
I missed that there were different package names.  This should be fine

**Comment by austinkelleher** [comment]
This has been fixed and published in `marko@4.0.0-rc.6`.
